BST/BST.cpp

```diff
@@ -0,0 +1,310 @@
+#include<iostream>
+#include<queue>
```

Comment:
Delete this, we are not yet on BST. When we come on that, then commit this code.